OpenOversight/app/main/views.py

```diff
@@ -673,7 +673,7 @@ def list_officer(
     ):
         form_data["gender"] = genders
 
-    unit_choices = [
+    unit_choices = ["Not Sure"] + [
```

Comment:
Why was this change necessary? It looks like it'd produce the same results 🤔

Reply:
Not exactly! "Not Sure" used to be added to the unit options list further down in the code:
https://github.com/OrcaCollective/OpenOversight/blob/ca674ef880b91f6df1738287b3b13e76e94f270f/OpenOversight/app/main/views.py#L718

But that meant that `form_data["unit"]` wouldn't be set if "Not Sure" was selected since it wouldn't be in `unit_choices`. (`form_data["unit"]` is used later on to construct the search query and re-populate the input fields)
https://github.com/OrcaCollective/OpenOversight/blob/ca674ef880b91f6df1738287b3b13e76e94f270f/OpenOversight/app/main/views.py#L690-L693

And that's basically what caused all options to become deselected if "Not Sure" was selected (like in the "Before" screenshot)